pkg/keys/rsa.go

```diff
@@ -107,7 +112,9 @@ func (s *rsaSigner) SignMessage(message []byte) ([]byte, error) {
 }
 
 func (s *rsaSigner) ContainsID(id string) bool {
-	return s.PublicData().ContainsID(id)
+	publicData := s.PublicData()
```

Comment:
Leave this how it was

Reply:
I don't remember making this change. maybe `origin/master` changed?